.github/workflows/test.yml

```
@@ -0,0 +1,22 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip setuptools
+      - name: Install dependencies
+        run: |
+          pip install 'tutor[dev]>=16.0.0,<17.0.0'
```

Given that we "rely" on the Tutor Core testing tools, I wonder if adding the following section to the `setup.py`:

```
    extras_require={
        'dev': [
            'tutor[dev]>=16.0.0,<17.0.0'
        ]
    },
```
would be a bad practice. Writing the Tutor core version explicitly here requires changing this version in the next migration, but defining it in the `setup.py` is more maintainable IMHO. This line would become something like:
 
```suggestion
          pip install -e .[dev]
```

cc @regisb

---

That's an idea!

---

I think that would be great. But maybe it's inconvenient for @CodeWithEmad to open so many PRs? If necessary I can make the change myself for all plugins.

---

I'm okay with that. I'll open a new PR for all plugins after the `Testing and Linting` PRs get merged with them. is that okay?

---

I'm OK with that approach, so I proceed with the approval and merge, LGTM.